util/git/client.go

```diff
@@ -405,6 +406,43 @@ func (m *nativeGitClient) LsRemote(revision string) (res string, err error) {
 	return
 }
 
+// LsRemote resolves the commit SHA of a specific branch, tag, or HEAD. If the supplied revision
```

Comment:
Nit pick: first word of this comment should be `LsRemoteGitCLI` to match the name of the method